packages/monaco/src/browser/monaco-diff-navigator-factory.ts

```diff
@@ -29,29 +29,33 @@ export class MonacoDiffNavigatorFactory {
         hasPrevious: () => false,
         next: () => { },
         previous: () => { },
-        revealFirst: false,
     };
 
-    createdDiffNavigator(editor: IStandaloneDiffEditor, options?: IDiffNavigatorOptions): DiffNavigator {
-        const navigator = monaco.editor.createDiffNavigator(editor, options);
+    createdDiffNavigator(editor: IStandaloneDiffEditor | Monaco.editor.IStandaloneDiffEditor, options?: Monaco.editor.IDiffNavigatorOptions): DiffNavigator {
```

Comment:
Not quite sure why we need the union type here.

Reply:
It's a workaround for the internal/external type incomparability problem. This allows us to call this function from places that use either public or private imports.